source/extensions/config_subscription/grpc/grpc_mux_impl.cc

```diff
@@ -80,59 +84,61 @@ GrpcMuxImpl::GrpcMuxImpl(GrpcMuxContext& grpc_mux_context, bool skip_subsequent_
 
 std::unique_ptr<GrpcStreamInterface<envoy::service::discovery::v3::DiscoveryRequest,
                                     envoy::service::discovery::v3::DiscoveryResponse>>
-GrpcMuxImpl::createGrpcStreamObject(GrpcMuxContext& grpc_mux_context) {
+GrpcMuxImpl::createGrpcStreamObject(Grpc::RawAsyncClientPtr async_client,
```

Comment:
nit: when move semantics is intended prefer to declare arg types as rvalue references, e.g. `Grpc::RawAsyncClientPtr&&` etc

Reply:
I think the Envoy style guide allows both, but I understand the motivation behind this.
Updated.